.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+    -   id: fix-byte-order-marker
+    -   id: check-case-conflict
+    -   id: check-merge-conflict
+    -   id: check-symlinks
+    -   id: destroyed-symlinks
+    -   id: check-vcs-permalinks
+    -   id: detect-private-key
+    -   id: no-commit-to-branch
+
+-   repo: https://github.com/jvstein/pre-commit-dotnet-format
+    rev: 'v0.2'
```

Comment:
well this PR was open for long enough for this [project to get deprecated](https://github.com/jvstein/pre-commit-dotnet-format#deprecated)

Reply:
Looks like the deprecating project does not yet have a release, and the head branch depends on .NET 6 preview.